### DISCUSSION TITLE
Comparison to similar frameworks (unjs/nitro)?

### DISCUSSION THREAD

**Discussion by septatrix** [honojs/hono#1988] [General]
Does anyone have an overview how this compares to unjs/nitro, the server powering Nuxt.js?
Are there plans to maybe have a comparison table on the website?

There is already #1166 though that asks about performance. I would be more interested in features like Websocket support, streaming responses, automatic schema generation (OpenAPI) etc.

**Comment by yusukebe** [comment]
That's a great idea, I'll do it after v4 is released or have someone else do it.
